.github/workflows/wikiman.yml

```diff
@@ -0,0 +1,21 @@
+name: wikiman
+
+on:
+  schedule:
+    - cron:  '6 12 * * *'
```

Comment:
just a random time, see the notice about delays here https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#scheduled-events

probably could be 0 0 or something just as well.

Reply:
I might add this as a comment next to the cron!